sysml.library/Domain Libraries/Geometry/ShapeItems.sysml

```diff
@@ -35,7 +23,20 @@ package ShapeItems {
 		attribute :>> outerSpaceDimension = 2;
 	}
 
-	item def Circle :> Shell {
+
+	/**
+	 * Most general surface shape.	Solids are objects with these as space boundary.	
+	 */
+	abstract item def Shell :> Item, StructuredSpaceObject, Surface	 {
+	}
+
+	/**
+	 * Most general curve shape.	Solids are objects with these as space boundary.	
```

Comment:
Don't solids have 2D Shells as space boundary, not 1D Paths?

Reply:
Cr*p, thx.  Will fix in push.  Item::isSolid=true requires them to be dim 3 (I checked the definition with HP).